src/features/metadata-instance-editor/Instance.js

```diff
@@ -530,12 +534,16 @@ class Instance extends React.PureComponent<Props, State> {
      * @return {boolean} true if editable
      */
     canEdit(): boolean {
-        const { canEdit, onModification, onRemove, onSave }: Props = this.props;
-        return (
+        const { canEdit, onModification, onRemove, onSave, template, isCascadingPolicyApplicable }: Props = this.props;
+        const isCustom = template.templateKey === TEMPLATE_CUSTOM_PROPERTIES;
+        const hasCascadePermission = isCustom ? true : isCascadingPolicyApplicable;
```

Comment:
wouldn't this prevent editing metadata on files?

Reply:
You're right that it does. This should not be the case. So, need to figure out how to get whether the template is on a file or folder.

Template has a property called “typeKey,” which is always an empty string when inspecting metadata on a folder and is set to “metadata_folder” when looking at metadata on a file. Although I’ve come across some instances of this being used in the monolith codebase, it’s not very clear on the specific conditions in which this is set. Therefore, I’m not 100% confident that this alone will effectively differentiate between folders and files when rendering the metadata.

Alternatively, I could pass in "who called" as 'isFromFolder', for example. I would rather not do this though